packages/salesforcedx-vscode-core/package.json

```diff
@@ -144,6 +144,11 @@
           "when": "view == metadata && viewItem =~ /(folder|type)/",
           "group": "inline"
         },
+        {
+          "command": "sfdx.force.source.retrieve.open.component",
+          "when": "view == metadata && viewItem =~ /(type|component)/",
```

Comment:
@smaddox-sf should there be a retrieve & open button on type nodes? Would the behavior be to open a file for every component that was retrieved?

Reply:
I have changed that condition to apply only on files, let me know if it's OK